### DISCUSSION TITLE
Changing existing contact from Group to Private cause uploading to fail.

### DISCUSSION THREAD

**Discussion by vk1kcm** [hmatuschek/qdmr#332] [General]
I was updating the codeplug on my rt3s to the latest VKDMR standard.  I already had a contact, channel and zone definition for TG9990.  The contact had this defined as a group call.  I changed this to private call and tried to upload the codeplug and it failed with an error message that said "contact must be a group call".  Creating a new contact and channel worked fine.

**Comment by hmatuschek** [comment]
This contact was added to some group list. These group lists can only contain group calls on many radios. Actually on all radios except those running the OpenGD77 codeplug. Just check your group lists and it should work again.

**Comment by allesand** [comment]
Link to #273 - similar issue (also https://github.com/hmatuschek/qdmr/discussions/219 )

A talkgroup is always a group call, is it not? Just trying to find out how things might be used differently - the "quick access with OpenGD77" being the only reason to be found so far.
